src/localization/bundles/en.json

```diff
@@ -714,9 +716,15 @@
-    "server.addserver.installos.cancel.confirm": "This will cancel the OS installation for the new compute server(s). \n\nDo you want to continue? ",
+    "server.deploy.failure.cancel.confirm": "Cancelling could potentially leave the cloud deployment in a half state. It will close the window, and any error messages will no longer be available to view.\n\nDo you want to cancel?",
+    "server.addserver.failure.prep.cancel.confirm": "Cancelling will close the window, and any error messages will no longer be available to view. \n\nThe following steps need to be performed manually on the cloud deployer before clicking Yes to cancel... \n1. cd ~/openstack/my_cloud/definition/data \n2. Remove the new compute server(s) from servers.yml \n3. git commit -a -m 'commit message' \n4. cd ~/openstack/ardana/ansible \n5. ansible-playbook -i hosts/localhost config-processor-run.yml \n6. ansible-playbook -i hosts/localhost ready-deployment.yml  \n\n If the Add Server page continues to display the removed servers, refresh the page using the web browser. To retry deployment, re-add the servers to the appropriate role, and click the \"Deploy\" button. \n\n Do you want to cancel?",
+    "server.addserver.failure.add.cancel.confirm": "Cancelling could potentially leave the process of adding the new compute server(s) in a half state. It will close the window, and any error messages will no longer be available to view. \n\nThe following steps need to be performed manually on the cloud deployer before clicking Yes to cancel... \n1. cd ~/openstack/my_cloud/definition/data \n2. Remove the new compute server(s) from servers.yml \n3. git commit -a -m 'commit message' \n4. cd ~/openstack/ardana/ansible \n5. ansible-playbook -i hosts/localhost config-processor-run.yml \n6. ansible-playbook -i hosts/localhost ready-deployment.yml  \n\n If the Add Server page continues to display the removed servers, refresh the page using the web browser. To retry deployment, re-add the servers to the appropriate role, and click the \"Deploy\" button. \n\n Do you want to cancel?",
+    "server.addserver.installos.cancel.confirm": "This will cancel the OS installation for the new compute server(s).  \n\nDo you want to cancel?",
+    "server.replace.compute.osinstall.failure.cancel.confirm": "Cancelling wil close the window and any error messages will no longer be available to view. \nAfter cancel, the new and undeployed compute server {0} shows up in servers Summary page, go to Add server page and unassign it from the role before replace compute server {1} again.  \n\nDo you want to cancel?",
```

Comment:
the middle part of this should read:
`After canceling, the new and undeployed compute server {0} will show up in the Server Summary page. Navigate to the Add Server page to unassign it from it's role before attempting to replace server {1} again`

Reply:
It might also be good to replace `Do you want to cancel?` with `Proceed with cancellation?`

Reply:
also `will` is mis-spelled `wil` here

Reply:
@jeremy-moffitt  ok , will update